registry_inspector/inspector.py

```diff
@@ -1,4 +1,4 @@
-from registry_inspector.registry_client import RegistryClient
+from registry_client import RegistryClient
```

Comment:
The `from registry_inspector.registry_client import ...` is correct here. Implicit relative imports cause all sorts of problems.